README.md

```diff
@@ -3,27 +3,31 @@
 In this exercise you will work with your group to get to full code coverage and fix any bugs that you find.
 
 ## Setup
-1. Clone a copy of this exercise. This command makes a new folder called `code-coverage-exercise`, and then puts the exercise into this new folder. You do not need to fork the repository.
+1. In Github, click on the "Fork" button in github and fork the repository to your Github account. This will make a copy of the project in your github account.
```

Comment:
```suggestion
1. On [Github](https://github.com/AdaGold/code-coverage-exercise), click on the "Fork" button in the top right corner and fork the repository to your Github account. This will make a copy of the project in your Github account.
```